lib/core/package.json

```diff
@@ -20,8 +20,8 @@
   "files": [
     "dist/**/*",
     "dll/**/*",
+    "prebuilt/**/*",
     "types/**/*",
-    "README.md",
```

Comment:
Why the removal of the README.md? this is a pretty common practice for npm packages.